go.mod

```diff
@@ -206,6 +206,7 @@ require (
 	github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
 	github.com/facebookgo/subset v0.0.0-20200203212716-c811ad88dec4 // indirect
 	github.com/facebookincubator/flog v0.0.0-20190930132826-d2511d0ce33c // indirect
+	github.com/fatih/camelcase v1.0.0 // indirect
```

Comment:
That's interesting. no other deps were changed and we have new indirect one?

Reply:
I guess some new feature is now used from some already existing direct dependency:
```console
$ go mod why github.com/fatih/camelcase
# github.com/fatih/camelcase
github.com/stackrox/rox/roxctl/common
k8s.io/kubectl/pkg/polymorphichelpers
k8s.io/kubectl/pkg/describe
github.com/fatih/camelcase
```